Gemfile.lock

```diff
@@ -237,72 +238,72 @@ GEM
       timeout
     net-protocol (0.1.3)
       timeout
-    net-scp (3.0.0)
-      net-ssh (>= 2.6.5, < 7.0.0)
+    net-scp (4.0.0.rc1)
+      net-ssh (>= 2.6.5, < 8.0.0)
     net-smtp (0.3.1)
       digest
       net-protocol
       timeout
-    net-ssh (6.1.0)
+    net-ssh (7.0.1)
     nio4r (2.5.8)
-    nokogiri (1.13.3-x86_64-darwin)
+    nokogiri (1.13.8-x86_64-darwin)
       racc (~> 1.4)
     orm_adapter (0.5.0)
     pagy (5.10.1)
       activesupport
     parallel (1.22.1)
-    parser (3.1.1.0)
+    parser (3.1.2.1)
```

Comment:
### [parser](https://github.com/whitequark/parser)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.1.1.0 → 3.1.2.1

[_[change-log](https://github.com/whitequark/parser/blob/v3.1.2.1/CHANGELOG.md), [source-code](https://github.com/whitequark/parser/tree/v3.1.2.1), [gem-diff](https://my.diffend.io/gems/parser/3.1.1.0/3.1.2.1)_]





<details>
<summary>Commits</summary>

A change of **10** commits. See the full changes on [the compare page](https://github.com/whitequark/parser/compare/v3.1.1.0...v3.1.2.1).

These are the individual commits:
- (ec523c0) [Update changelog.](https://github.com/whitequark/parser/commit/ec523c0f8e9eaedcfb892b8e64394efbce9777d8)
- (934bd00) [Upgrade Github Actions test workflow (#849)](https://github.com/whitequark/parser/commit/934bd0013622d2eceffd58dcfc63feba6844d7f4)
- (1d275fc) [- ruby32.y: fix using p_kwnorest in f_no_kwarg (#848)](https://github.com/whitequark/parser/commit/1d275fc21227f538a2dda74a54939cd55b1c9200)
- (ecdcfdd) [* Bump maintenance branches to 3.1.2, 3.0.4, 2.7.6, and 2.6.…](https://github.com/whitequark/parser/commit/ecdcfddc4e7b7506e862b39209cb415050105ba7)
- (c5f26c3) [Bump version.](https://github.com/whitequark/parser/commit/c5f26c35cd6904bf6007245ab1e66416cadb8ccd)
- (4597f27) [Update changelog.](https://github.com/whitequark/parser/commit/4597f27f42163ee6aea441c7758c8e410e92770d)
- (4928714) [Update changelog.](https://github.com/whitequark/parser/commit/492871419b298e233bb788ec1793b7af7004e23e)
- (27f83ba) [- ruby31.y: fix preparing the state for one-line patterns wi…](https://github.com/whitequark/parser/commit/27f83bab58d4f44dba7e20f7b46920044e0a2365)
- (029ea4c) [Bump version.](https://github.com/whitequark/parser/commit/029ea4c3e54648295a41c30f566c9b2e4afdb2cd)
- (d0b22cd) [Update changelog.](https://github.com/whitequark/parser/commit/d0b22cd2f255c89b164b0958d698fdb4f0eb3f8e)


</details>